### PR TITLE
Add pr template for backport and other things

### PR DESCRIPTION
**Pull request by MalloZup**
this fix a lack of knowledge

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## What does this PR change?
+
+**add description**
+( Previous behaviour, Current behaviour etc).
```
Comment (drpaneas) [inline .github/PULL_REQUEST_TEMPLATE.md]: * Remove the ` ` between `(` and `P`.
* Remove the `etc` or write that `etc` means,.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Backport to other branches
+Use  git cherry-pick -x (or a variant of it) to backport a commit from master to branches
+
+- [ ] release-2.1 
+- [ ] release-3.0
+
+## Documentation
```
Comment (drpaneas) [inline .github/PULL_REQUEST_TEMPLATE.md]: Add an example of a backport as a guideline.

Comment (MalloZup) [reply .github/PULL_REQUEST_TEMPLATE.md]: i don't think we need this

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (MalloZup) [reply .github/PULL_REQUEST_TEMPLATE.md]: ok

Comment (MalloZup) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

**Comment by MalloZup** [comment]
:tada: i hope this ci will be green without flaky issue.. :pray:

**Pull request merged**